### PR TITLE
Add helpers to utilities bundle

### PR DESCRIPTION
**Pull request by mjansing**
Bootstrap bundles utility classes in a dedicated bootstrap-utilities deliverable (`bootstrap-utilities.css`) but the deliverable does not contain all utilities mentioned in the documentation (e.g. `.sticky-top`, `.stretched-link` or `.text-truncate`). These missing utilities are located in `scss/helpers/` and currently not part of it. This PR adds them.

### CODE REVIEW

**Review by mdo — APPROVED** [review]
I'm into it. Feels like a natural fit.

**Comment by ffoodd** [comment]
Shouldn't it be mentionned in the docs?

**Comment by XhmikosR** [comment]
@ffoodd like where? It seems utilities are already mentioned in https://getbootstrap.com/docs/5.0/getting-started/contents/#css-files. Feel free to push to this branch if you have any ideas.

**Comment by ffoodd** [comment]
Don't really know, maybe we didn't mention helpers anywhere before?

It should at least appear in the migration guide since people importing only our utilities in a custom build will now import  helpers incidentally, what they might have done explicitly before.

**Comment by XhmikosR** [comment]
@mjansing can you please take care of the above comment?

**Comment by mjansing** [comment]
@XhmikosR I added a migration note.

File: site/content/docs/5.0/migration.md [hunk 1]
```diff
@@ -11,7 +11,10 @@ toc: true
 
 ### Sass
 
+#### Utilities
+
 - Extended the `.visually-hidden-focusable` helper to also work on containers, using `:focus-within`.
+- `bootstrap-utilities.css` now also includes our helpers. Helpers don't needs to be imported in custom builds anymore.
```
Comment (ffoodd) [inline site/content/docs/5.0/migration.md:17]: Still an extra letter: need**s** 🙂

**Review by ffoodd — DISMISSED** [review]

**Review by MartijnCuppens — APPROVED** [review]

File: site/content/docs/5.0/migration.md (hunk 1 above)
Comment (mjansing) [reply site/content/docs/5.0/migration.md]: I'm sorry.

**Review by ffoodd — APPROVED** [review]

**Review by mdo — APPROVED** [review]
LGTM!

**Pull request merged**